alwaysontop/render.js

```diff
@@ -304,19 +312,13 @@ class AlwaysOnTop extends EventEmitter {
             }
         };
 
-        // Change the dom of 'about:blank' to display the always on top content.
-        this._alwaysOnTopWindow.onload = () => {
-            if (!this._alwaysOnTopWindow) {
-                return;
-            }
-
             const cssPath = path.join(__dirname, './alwaysontop.css');
             const jsPath = path.join(__dirname, './alwaysontop.js');
 
             // Add the markup for the JS to manipulate and load the CSS.
             this._alwaysOnTopWindow.document.body.innerHTML = `
               <div id="react"></div>
-              <video autoplay="" id="video" style="transform: none;"></video>
+              <video autoplay="" id="video" style="transform: none;" muted="muted"></video>
```

Comment:
I don't think a value is necessary for the `muted` property

Reply:
Will try without it.